doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -808,34 +808,35 @@ combined to create custom rewriting procedures. Its set of strategies is based
 on the programmable rewriting strategies with generic traversals by Visser et al.
 :cite:`Luttik97specificationof` :cite:`Visser98`, which formed the core of
 the Stratego transformation language :cite:`Visser01`. Rewriting strategies
-are applied using the tactic :n:`rewrite_strat @rewstrategy`.
+are applied using the :tacn:`rewrite_strat` tactic`.
 
-.. insertprodn rewstrategy rewstrategy
+.. insertprodn rewstrategy rewstrategy0
 
 .. prodn::
-   rewstrategy ::= @one_term
-   | <- @one_term
```

Comment:
IIUC this production is entirely gone.  Doesn't that present a compatibility issue?